pkg/kube/kubehttp.go

```diff
@@ -28,15 +28,15 @@ import (
 func CreateScheme() (*apiruntime.Scheme, error) {
 	scheme := apiruntime.NewScheme()
 	builder := apiruntime.SchemeBuilder{
-		sourcev1beta2.AddToScheme,
-		kustomizev1beta2.AddToScheme,
+		sourcev1.AddToScheme,
+		kustomizev2.AddToScheme,
```

Comment:
1. assume only v1 -> we could support flux v2 but we break < 0.4x
release this 
do we have customers asking for backward compatibility ? 
in case yes, we could open a ticket to think on supporting backward compatibility as follow up